test/helpers/utils.js

```diff
@@ -25,6 +25,12 @@ export const createFakeInterface = value => ({
 	},
 });
 
+export const createFakeBrokenInterface = () => ({
+	on: () => {
+		return createFakeInterface();
```

Comment:
This would make more sense as `createFakeBrokenInterface`, otherwise calling `.on` can fix it!